cookbooks/rubygems-sensu/recipes/app.rb

```diff
@@ -26,3 +26,13 @@
   interval 30
   additional(occurrences: 3)
 end
+
+%w( rubygems.org fastly.rubygems.org rubygems.global.ssl.fastly.net ).each do |host|
+  sensu_check "check_#{host}_ssl" do
+    command "perl /etc/sensu/plugins/check_postgres_backends.rb -P '' -h #{host}"
```

Comment:
`-P` option needs path to rubygems installation and I don't know where exactly is that on server.